d3a_api_client/websocket_device.py

```diff
@@ -0,0 +1,80 @@
+import asyncio
+import logging
+import json
+import threading
+import traceback
+import websockets
+from d3a_interface.utils import wait_until_timeout_blocking
+
+
+class RestWebsocketAPIException(Exception):
+    pass
+
+
+class WebsocketMessageDispatcher:
```

Comment:
As discussed offline, the websocket in the API is used for sending data, and in my perception, on this end, it really receives the data, so IMO it should be renamed to WebsocketMessageReceiver. Or did i get it wrong?

Reply:
Indeed you are right, naming you suggest is much better. Fixing this